client/router.js

```diff
@@ -25,23 +25,23 @@ FlowRouter.route('/', {
     }
   ],
   action: function(params) {
-    BlazeLayout.render('HomeLayout', { header: "header", main: "home" });
+    BlazeLayout.render('HomeLayout', {main: "home" });
   }
 });
 
 FlowRouter.route('/cadastro/estudante', {
   name: 'studentSignup',
   title: "Banco de Talentos - Cadastrar Aluno",
   action: function(params) {
-    BlazeLayout.render('HomeLayout', { header: "header", main: "studentSignup" });
+    BlazeLayout.render('HomeLayout', {main: "studentSignup" });
   }
 });
 
 FlowRouter.route('/cadastro/empresa', {
   name: 'companySignup',
   title: "Banco de Talentos - Cadastrar Empresa",
   action: function(params) {
-    BlazeLayout.render('HomeLayout', { header: "header", main: "companySignup" });
+    BlazeLayout.render('HomeLayout', {main: "companySignup" });
```

Comment:
As explained in the other comment, we changed the way we renderm now we render the template 'header' like this:

```html
{{> header }}
```

Then we don't need to tell Blaze which template he should render, he already knows it